core/src/main/resources/i18n/displayStrings.properties

```
@@ -3006,12 +3006,12 @@ password.deriveKey=Derive key from password
 password.walletDecrypted=Wallet successfully decrypted and password protection removed.
 password.wrongPw=You entered the wrong password.\n\nPlease try entering your password again, carefully checking for typos or spelling errors.
 password.walletEncrypted=Wallet successfully encrypted and password protection enabled.
-password.walletEncryptionFailed=Wallet password could not be set. You might have imported seed words which are not matching the wallet database. Please contact the developers on the Bisq Forum.
+password.walletEncryptionFailed=Wallet password could not be set. You may have imported seed words which do not match the wallet database. Please contact the developers on Keybase - 'https://keybase.io/team/bisq' .
```

```suggestion
password.walletEncryptionFailed=Wallet password could not be set. You may have imported seed words which do not match the wallet database. Please contact the developers on Keybase (https://keybase.io/team/bisq).
```
I don't like the quotes, but this might be subjective.

---

Enclosing the URL as follows will make it clickable, `[HYPERLINK:https://keybase.io/team/bisq]`
There are other example in displayStrings.properties.

---

Wonderful! Not sure how I missed that.

---

Ya not sure why I used quotes there. Thanks!